packages/teams-js/src/public/app.ts

```diff
@@ -701,6 +701,13 @@ export namespace app {
      * Will be `undefined` when not running in Teams.
      */
     team?: TeamInfo;
+
+    /**
+     * When `processActionCommand` activates a dialog, this dialog should automatically fill in some fields with information. This information comes from M365 and is given to `processActionCommand` as `extractedParameters`.
+     * App developers need to use these `extractedParameters` in their dialog.
+     * They help pre-fill the dialog with necessary information (`dialogParameters`) along with other details.
+     */
+    dialogParameters?: Record<string, string>;
```

Comment:
We also need to add the parameter in legacyContext. hubSDK sends over the legacy context, and teams-js change it to the app context. I will show you an example in hubSDk

Reply:
Updated! Thanks for letting me know the legacyContext!!